src/libs/mrubyWriterConnector.ts

```diff
@@ -9,7 +9,7 @@ type Listener = (buffer: string[]) => void;
 
 type Reader = ReadableStreamDefaultReader<Uint8Array>;
 type Writer = WritableStreamDefaultWriter<Uint8Array>;
-type Event = "AttemptToEnterWriteMode" | "SuccessToExitWriteMode";
+type Event = "onEnterWriteMode" | "SuccessToExitWriteMode";
```

Comment:
```suggestion
type Event = "SuccessToEnterWriteMode" | "SuccessToExitWriteMode";
```
`SuccessToExitWriteMode`に合わせたほうがよいでしょう。